lib/archethic/bootstrap/network_init.ex

```diff
@@ -99,6 +99,14 @@ defmodule ArchEthic.Bootstrap.NetworkInit do
     Transaction.new(
       :origin_shared_secrets,
       %TransactionData{
+        code: """
```

Comment:
Good for the code contracts, just I don't think we need to constraints to the origin family here, this will mean even the node which are doing the transaction will have to use an origin private key generated from software.
So nodes might use hardware key as proof of enrollment to leverage the POW, so I think you can omit it.